### DISCUSSION TITLE
Why can't I add an integration to my account?

### DISCUSSION THREAD

**Discussion by s-theo** [vercel/community#4561] [Help] [answered]
### Page to Investigate

https://vercel.com/integrations

### Steps to Reproduce

When I want to install [checkly](https://vercel.com/integrations/checkly) for my account,

The following situations occur,
<img width="672" alt="image" src="https://github.com/vercel/community/assets/89436345/b96186cf-16ea-4a8e-9aeb-e5183b4feb37">

And there is no way to select a user or team.

**Comment by s-theo** [comment]
The same problem occurs when I want to install other integrations.

Is this because I don't have enough permissions?

Or some options are not enabled in other settings?

**Comment by swarnava** [comment]
Hi,

Which web browser are you using?

**Comment by s-theo** [comment, marked as answer]
@swarnava Thank you for your reply! I have installed it. The reason is that my browser plug-in makes the page not work, but I can install it after I turn off the plug-in.
